roles/sideload_kernel/tasks/main.yml

```diff
@@ -0,0 +1,124 @@
+---
+- name: Validate required variables
+  ansible.builtin.assert:
+    that:
+      - sideload_kernel_uri | string
+
+- name: Install kubernetes.core dependencies
+  block:
+    - name: Prefer package manager
+      become: true
+      ansible.builtin.package:
+        name: python3-kubernetes
+        state: present
+  rescue:
+    - name: Fallback to pip installation
+      ansible.builtin.pip:
+        name: kubernetes
+
+- name: Sideload-kernel job namespace
+  kubernetes.core.k8s:
+    state: present
+    api_version: v1
+    kind: Namespace
+    name: "{{ sideload_kernel_namespace }}"
+  environment: "{{ k8s_auth }}"
+
+- name: Sideload-kernel configmap
+  kubernetes.core.k8s:
+    state: present
+    apply: true
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: sideload-kernel
+        namespace: "{{ sideload_kernel_namespace }}"
+      data:
+        KERNEL_URI: "{{ sideload_kernel_uri }}"
+        flip_kernel: "{{ lookup('file', 'flip_kernel') }}"
+  environment: "{{ k8s_auth }}"
+  register: configmap_created
+
+- name: Clean out any previous jobs
+  kubernetes.core.k8s:
+    state: absent
+    api_version: batch/v1
+    kind: Job
+    name: flip-kernel
+    namespace: "{{ sideload_kernel_namespace }}"
+  environment: "{{ k8s_auth }}"
+  when: configmap_created.changed or sideload_kernel_force | bool
+
+- name: Sideload-kernel job
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: flip-kernel
+        namespace: "{{ sideload_kernel_namespace }}"
+      spec:
+        backoffLimit: 3
+        # TODO: Add a selector for multi-node cluster cases?
+        template:
+          spec:
+            containers:
+              - command:
+                  - /bin/bash
+                args:
+                  - "-c"
+                  - "install /script/flip_kernel /host/tmp && chroot /host /tmp/flip_kernel ${KERNEL_URI}"
+                env:
+                  - name: KERNEL_URI
+                    valueFrom:
+                      configMapKeyRef:
+                        name: sideload-kernel
+                        key: KERNEL_URI
+                image: ubi9
```

Comment:
Can you make this one a variable with ubi9 being the default? this way we can make it work in a disconnected environment by passing the Fully Qualified Repository Name of the local registry